pyproject.toml

```diff
@@ -1,25 +1,29 @@
 [tool.poetry]
-name = <$package!quote$>
-version = <$inital_version!quote$>
-description = <$description!quote$>
-authors = <$authors!quote_each$>
-license = <$license!short!quote$>
+name = "heksher"
+version = "0.1.0.dev"
+description = ""
+authors = ["Biocatch LTD <serverteam@biocatch.com>"]
+license = "MIT"
 readme = "README.md"
-repository = <$repo_link!quote$>
+repository = "https://github.com/biocatchltd/heksher-py"
 packages = [
-    {include=<$package!quote$>}
+    {include="heksher"}
 ]
 
 [tool.poetry.dependencies]
```

Comment:
we probably need to be way more liberal on the dependencies versions, as we don't want to restrict users. 
I'd set it for same major as minimum